content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -216,6 +216,22 @@ Used on: Service
 
 Kubernetes uses this label to differentiate multiple Services. Used currently for `ELB`(Elastic Load Balancer) only.
 
+### kubernetes.io/service-account.name
+
+Example: `kubernetes.io/service-account.name: "sa-name"`
+
+Used on: Secret
+
+This annotation is used to set service account name by which the token identifies.
+
+### kubernetes.io/service-account.uid
+
+Example: `kubernetes.io/service-account.uid: da68f9c6-9d26-11e7-b84e-002dc52800da`
+
+Used on: Secret
+
+This annotation is used to set the UID of the service account by which the token identifies.
```

Comment:
Extra, recommended tweak (because some people expand “UID” to “User Identifier”) - they might thing that ServiceAccounts have a user id, rather than that _all objects_ have a unique ID.

```suggestion
This annotation records the {{< glossary_tooltip term_id="uid" text="unique ID" >}} of the
ServiceAccount that the token (stored in the Secret) represents.
```